### DISCUSSION TITLE
Streamlink Twitch GUI stopped working: unexpected version check output

### DISCUSSION THREAD

**Discussion by heidiwenger** [streamlink/streamlink-twitch-gui#877] [Q&A] [answered]
### Checklist

- [X] I understand that **Streamlink Twitch GUI** is just a launcher for **Streamlink**
- [X] [I have read the contribution guidelines](https://github.com/streamlink/streamlink-twitch-gui/blob/master/CONTRIBUTING.md)
- [X] [I have checked the list of open and recently closed issues](https://github.com/streamlink/streamlink-twitch-gui/issues?q=is%3Aissue+sort%3Aupdated-desc)
- [X] [I have checked the commit log of the master branch](https://github.com/streamlink/streamlink-twitch-gui/commits/master)

### Streamlink Twitch GUI version

2.0.0

### Streamlink version

3.1.1-2-cp310-cp310-manylinux2014

### Operating system, environment and configuration details

Ubuntu 20.04 LTS

### Description

The used assets are the latest: 
streamlink-3.1.1-2-cp310-cp310-manylinux2014_x86_64.AppImage
streamlink-twitch-gui-v2.0.0-x86_64.AppImage

Streamlink executable is set correctly: /home/user/Downloads/streamlink-3.1.1-2-cp310-cp310-manylinux2014_x86_64.AppImage

No clue what to do to get watching Twitch again through streamlink-twitch-gui. Any help? Why did this happen?

Thanks.

**Comment by bastimeyer** [comment, marked as answer]
Execute
```sh
/home/user/Downloads/streamlink-3.1.1-2-cp310-cp310-manylinux2014_x86_64.AppImage --version
```
and post the entire output.

> unexpected version check output

This means that whatever you have told the Twitch GUI to use as streamlink executable doesn't return a valid version string when checking the Streamlink version. This can happen when it returns an error message instead. So do as I mentioned above and check the output.
https://github.com/streamlink/streamlink-twitch-gui/wiki/Streamlink-configuration#unexpected-version-check-output-error
